oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/EDTFDatePickerField/EDTFDateRangePickerField.jsx

```diff
@@ -0,0 +1,208 @@
+            required={false}
+          />
+        </Form.Field>
+        {showSingleDatePicker ? (
+          <div>
+            <EDTFDatePickerWrapper
+              fieldPath={fieldPath}
+              handleChange={handleSingleDateChange}
+              handleClear={handleClearSingleDate}
+              placeholder={singleDateInputPlaceholder}
+              dateEdtfFormat={dateEdtfFormat}
+              setDateEdtfFormat={setDateEdtfFormat}
+              dateFormat={dateFormat}
+              clearButtonClassName={clearButtonClassName}
+              datePickerProps={{ selected: startDate }}
+              helpText={helpText}
+            />
+          </div>
+        ) : (
+          <GroupField>
+            <EDTFDatePickerWrapper
+              fieldPath={fieldPath}
+              handleChange={handleStartDateChange}
+              handleClear={handleClearStartDate}
+              placeholder={startDateInputPlaceholder}
+              dateEdtfFormat={dateEdtfFormat}
+              setDateEdtfFormat={setDateEdtfFormat}
+              dateFormat={dateFormat}
+              clearButtonClassName={clearButtonClassName}
+              datePickerProps={{
+                selected: startDate,
+                startDate: startDate,
+                endDate: endDate,
+                selectsStart: true,
+              }}
+              customInputProps={{ label: i18next.t("From") }}
+            />
+            <EDTFDatePickerWrapper
+              fieldPath={fieldPath}
+              handleChange={handleEndDateChange}
+              handleClear={handleClearEndDate}
+              placeholder={endDateInputPlaceholder}
+              dateEdtfFormat={dateEdtfFormat}
+              setDateEdtfFormat={setDateEdtfFormat}
+              dateFormat={dateFormat}
+              clearButtonClassName={clearButtonClassName}
+              datePickerProps={{
+                selected: endDate,
+                startDate: startDate,
+                endDate: endDate,
+                selectsEnd: true,
+                minDate: startDate,
+              }}
+              customInputProps={{ label: i18next.t("To") }}
+            />
+          </GroupField>
+        )}
+      </Form.Field>
+      <label className="helptext">{helpText}</label>
+    </React.Fragment>
+  );
+};
+
+EDTFDaterangePicker.propTypes = {
+  fieldPath: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+  icon: PropTypes.string,
+  helpText: PropTypes.string,
+  required: PropTypes.bool,
+  clearButtonClassName: PropTypes.string,
+  startDateInputPlaceholder: PropTypes.string,
+  endDateInputPlaceholder: PropTypes.string,
+  singleDateInputPlaceholder: PropTypes.string,
+};
+
+EDTFDaterangePicker.defaultProps = {
+  icon: "calendar",
+  helpText: i18next.t(
+    "Choose the time interval in which the event took place. If start and end date are the same, choose same date twice.You can also use the button above to select just the one date."
```

Comment:
Remove the "If start and end date...", it is now redundant thanks to the radio select